common/include/pcl/common/time.h

```diff
@@ -63,8 +63,6 @@ namespace pcl
       {
       }
 
-      /** \brief Destructor. */
-      virtual ~StopWatch () = default;
       //TODO: check if copy/move construcors/assignment operators are needed
```

Comment:
This can probably also be removed: it is a POD class, not really handling pointers or references.